pentasharp/Controllers/AccountController.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApplication1.Controllers
+{
+    public class AccountController : Controller
```

Comment:
Since this will be handling authentication processes, I would rename this AuthenticateController